app/models/project.rb

```diff
@@ -29,6 +30,10 @@ class Project < ActiveRecord::Base
     event :reject do
       transitions from: :proposed, to: :rejected, after: -> { self.comments_locked = true }
     end
+    
+    event :pending do
```

Comment:
I wonder if `pending` is the best name for the event. You introduced a trait for the projects factory called `review` below (see also my comment there), suggesting that you had a different name in mind earlier. What do you think about `start_review` as the event name (whilst keeping `pending` as the state)?